src/main/steps/common/task-list/components/notification-banner/config/da_applicant.ts

```diff
@@ -17,4 +19,13 @@ export const DA_APPLICANT: NotificationBannerProps[] = [
       return caseData?.state === State.CASE_CLOSED;
     },
   },
+  {
+    ...notificationBanner[BannerNotification.SOA_SERVED_DA],
+    show: (caseData: Partial<CaseWithId>, userDetails: UserDetails): boolean => {
+      return (
+        isCaseLinked(caseData, userDetails) &&
+        caseData.applicantsFL401?.response?.citizenFlags?.isApplicationToBeServed === YesOrNo.YES
```

Comment:
do defensive check for applicantsFL401 to ensure the code doesn't break.

Reply:
Added